sample_app/sampleapp_common.h

```diff
@@ -255,7 +300,44 @@ void app_print_network_details (
  * @return  0  if the operation succeeded.
  *          -1 if an error occurred.
  */
-int app_adjust_stack_configuration (pnet_cfg_t * stack_config);
+int app_pnet_cfg_init_default (pnet_cfg_t * stack_config);
+
+/**
+ * Initialize interface configuration from argument string.
+ *
+ * @param netif_list_str   In:    Comma separated list of network interfaces.
+ *                                Terminated string.
+ * @param p_cfg            InOut: p-net configuration
+ * @param verbosity        In:    Verbosity
+ * @return  0  on success
+ *         -1  on error
+ */
+int app_pnet_cfg_init_netifs (
+   const char * netif_list_str,
+   pnet_cfg_t * p_cfg,
+   int verbosity);
+
+/**
+ * Parse a comma separated list of network interfaces and check
+ * that the number of interfaces match the PNET_MAX_PORT configuration.
+ * Examples:
+ * PNET_MAX_PORT     arg_str                 status
+ * 1                 "eth0"                  ok
+ * 1                 "eth0,eth1"             err
+ * 2                 "br0,eth0,eth1"         ok
```

Comment:
Wouldn't this be an error? (three interfaces)

Reply:
For one physical port, we use that name.
For two physical ports, we use the two Linux Ethernet interfaces. But we must also create a third virtual Linux interface (a bridge). All three names must be supplied.

Reply:
The documentation in the code is correct. 
Multiport usage is described in \doc\multiple_ports.rst.
In this case "br0" will be the management port and "eth0" and "eth1" phyiscal ports.

Reply:
Maybe rename PNET_MAX_PORT to something like PNET_NUMBER_OF_PHYSICAL_PORTS for clarity (in another PR)?

Reply:
Good idea.